Makefile

```diff
@@ -656,3 +656,165 @@ endif
 .PHONY: build-must-gather
 build-must-gather: ## Build OADP Must-gather binary must-gather/oadp-must-gather
 	cd must-gather && go build -mod=mod -a -o oadp-must-gather cmd/main.go
+
+# Common AI review prompt
+define AI_REVIEW_PROMPT
+Review this git diff for a project called OADP (OpenShift API for Data Protection) operator. Focus on: \
```

Comment:
Start small please, this prompt is TOO general, and too big.
Let's get feedback from the team with regards to what would be smart to focus on first.

Even with 11G of ram provided, this is quite slow. 
IMHO a more effective first pass, MVP would be much faster and focus on one or two items of code quality.  IMHO the team should discuss and pick one or two items together.